src/cli/cli_udp.cpp

```diff
@@ -201,10 +329,33 @@ template <> otError UdpExample::Process<Cmd("linksecurity")>(Arg aArgs[])
 {
     otError error = OT_ERROR_NONE;
 
+    /**
+     * @cli udp linksecurity
+     * @code
+     * udp linksecurity
+     * Enabled
+     * Done
+     * @endcode
+     * @par
+     * Indicates whether link security is enabled or disabled.
+     */
     if (aArgs[0].IsEmpty())
     {
         OutputEnabledDisabledStatus(mLinkSecurityEnabled);
     }
+    /**
+     * @cli udp linksecurity(enable,disable)
```

Comment:
Space before the parenthesis, please

`@cli udp linksecurity (enable,disable)`